<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMultiInvoice</class>
 <widget class="QDialog" name="CreateMultiInvoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Multi Invoice</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Monthly Rent:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>No. Of Months:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QSpinBox" name="spinMonths"/>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="spinRent">
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Start Month:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="cboMonth">
       <item>
        <property name="text">
         <string>January</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>February</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>March</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>April</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>June</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>July</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>August</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>September</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>October</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>November</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dec</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Year:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="txtYear">
       <property name="text">
        <string>2014</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tenant Details:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Unit:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2" colspan="3">
      <widget class="QTreeWidget" name="trvUnits">
       <column>
        <property name="text">
         <string>Unit No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Company</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Property</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tenant Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Monthly Rent Amount</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QPlainTextEdit" name="txtTenantDetails">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblPreview">
     <column>
      <property name="text">
       <string>Month / Year</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rent Amount</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
